frontend/talentsearch/src/js/components/applicantProfile/ExperienceAndSkills.stories.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { Story, Meta } from "@storybook/react";
+import fakeExperiences, { generators } from "@common/fakeData/fakeExperiences";
+import ExperienceAndSkills, {
+  ExperienceAndSkillsProps,
+} from "./ExperienceAndSkills";
+
+export default {
+  component: ExperienceAndSkills,
+  title: "ApplicantProfile/ExperienceAndSkillsPage",
+} as Meta;
+
+const TemplateExperienceAndSkillsPage: Story<ExperienceAndSkillsProps> = (
+  args,
+) => {
+  return <ExperienceAndSkills {...args} />;
+};
+
+export const NoExperiences = TemplateExperienceAndSkillsPage.bind({});
+export const WithExperiences = TemplateExperienceAndSkillsPage.bind({});
+
+NoExperiences.args = {
+  experiences: [],
+};
+
+WithExperiences.args = {
+  experiences: fakeExperiences(10),
```

Comment:
The faker does not generate three levels intentionally (experiences -> skills -> experiences) to avoid a circular reference.  However, it makes the skills tab look bad.  I'm not sure what the solution would be.  Maybe a third story that uses the fakeSkills generator?

Reply:
I'm having trouble with this.. maybe you can assist me?

Reply:
Eh, I've been thinking about this and I think it's not a simple thing to solve.  It's just a faker so I recommend not worrying about it.  Let's let this one go.